<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>map_query</class>
 <widget class="QDialog" name="map_query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="Message_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>1451</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>1451</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起点</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="start">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>70</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>终点</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="end">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>70</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>10</y>
     <width>221</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QPushButton" name="load_map">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加载地图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清除地图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="shortest_rode">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>求两点间最短路径</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="all_rode">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>求两点间所有路径</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mintree">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>最小生成树</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
